docs/install.md

```diff
@@ -66,4 +66,18 @@ Follow the steps below to install Codacy on an existing Kubernetes cluster using
     [...]
     ```
 
-5.  Modify and use the [reference file](https://raw.githubusercontent.com/codacy/chart/master/codacy/values-production.yaml) `values-production.yaml` to [set up external DBs](configuration/external-dbs.md) (ideally running in a cloud managed Postgres instance).
+1. Download the [reference file](https://raw.githubusercontent.com/codacy/chart/master/codacy/values-production.yaml) `values-production.yaml` and configure the following values for each Codacy database:
```

Comment:
This part about using the `values-production.yaml` file and re-running Helm can still use some work!